var/spack/repos/builtin/packages/py-python-xlib/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPythonXlib(PythonPackage):
+    """The Python X Library is intended to be a fully
+    functional X client library for Python programs. It is
+    written entirely in Python, in contrast to earlier X
+    libraries for Python (the ancient X extension and the newer
+    plxlib) which were interfaces to the C Xlib."""
+
+    homepage = "https://github.com/python-xlib/python-xlib"
+    pypi     = "python-xlib/python-xlib-0.30.tar.gz"
+
+    version('0.30', sha256='74131418faf9e7b83178c71d9d80297fbbd678abe99ae9258f5a20cd027acb5f')
+
+    depends_on('python@2.7,3.3:3.6', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('python@2.7,3.3:', type=('build', 'run'))
```
The `setup.cfg` mentions Python 3.7 and 3.8 support, I'm guessing all newer versions will work.